content/docs/user-guide/experiment-management/experiments-overview.md

```diff
@@ -0,0 +1,75 @@
+# DVC Experiments Overview
+
+DVC Experiments are captured automatically by DVC when [run]. Each experiment
+creates and tracks a variation of your data science project based on the changes
+in your <abbr>workspace</abbr>.
+
+Experiments preserve a connection to the latest commit in the current branch
+(Git `HEAD`) as their parent or _baseline_, but do not form part of the regular
+Git tree or workflow (unless you make them [persistent]). This prevents
+polluting Git namespaces and bloating the repo unnecessarily.
+
+<details>
+
+### ⚙️ How does DVC track experiments?
+
+Experiments are custom [Git references](/blog/experiment-refs) (found in
+`.git/refs/exps`) with one or more commits based on `HEAD`. These commits are
+hidden and not checked out by DVC. Note that these are not pushed to Git remotes
+by default either (see `dvc exp push`).
+
+</details>
+
+[run them]: /doc/user-guide/experiment-management/running-experiments
+
+## Properties
+
+DVC Experiments will have an auto-generated name like `exp-bfe64` by default. A
+custom name can be given instead (using the `--name`/`-n` option of
+`dvc exp run`). These names can be used to reference experiments in other
+`dvc exp` subcommands.
+
+All experiments created by DVC will be associated to the latest commit (Git
+`HEAD`) at the time that they were run. This is called the experiment's
+_baseline_.
```

Comment:
Is this section needed? Naming details doesn't seem that important, and the baseline is already explained above.

Reply:
Hmm good points. I moved the naming paragraph to the hidden section for now (having it here instead of in the Running Exps guide was one of the original goals of the overview page). Removed the duplicate "baseline" explanation. Thanks